diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocMethods.kt

```diff
@@ -107,16 +89,14 @@ class KdocMethods(private val configRules: List<RulesConfig>) : Rule("kdoc-metho
                         ?.toSet() ?: setOf()
                 )
 
-        val paramCheckFailed = missingParameters.isNotEmpty() && !node.isSingleLineGetterOrSetter()
+        val paramCheckFailed = (missingParameters.isNotEmpty() || kDocParamList.isNotEmpty()) && !node.isSingleLineGetterOrSetter()
```

Comment:
The original logic was as follows: if there are undocumented parameters but a method is a simple setter, than there is no need to add KDoc for it's parameter. But if there already is a KDoc which has wtong parameter, you should fix it regardless of whether method is simple setter or not.